deploy/rbd/kubernetes/csi-rbdplugin-provisioner.yaml

```diff
@@ -100,8 +100,8 @@ spec:
             privileged: true
             capabilities:
               add: ["SYS_ADMIN"]
-          # for stable functionality replace canary with latest release version
-          image: quay.io/cephcsi/cephcsi:canary
+          # for stable functionality replace v2.1.0 with latest release version
```

Comment:
no need to update this comment, just keep it as it is. this applies to other places also

Reply:
done.